### PR TITLE
feat: sync v8.33.0

### PR DESCRIPTION
**Pull request by kecrily**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request?

sync v8.33.0

### CODE REVIEW

File: src/use/configure/language-options.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
 ---
-title: 语言选项
+title: 配置语言选项
 eleventyNavigation:
-    key: configuring language options
+    key: configure language options
     parent: configuring
```
Comment (mdjermanovic) [inline src/use/configure/language-options.md]:
```suggestion
    parent: configure
```

File: src/use/formatters/index.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ---
-title: 格式化工具
+title: 格式化工具参考
 eleventyNavigation:
     key: formatters
-    parent: user guide
+    parent: user eslint
```
Comment (mdjermanovic) [inline src/use/formatters/index.md]:
```suggestion
    parent: use eslint
```

File: src/contribute/development-environment.md [hunk 3]
```diff
@@ -1,14 +1,13 @@
 ---
-title: 开发环境
+title: 设置开发环境
 eleventyNavigation:
     key: set up a development environment
     parent: developer guide
```
Comment (mdjermanovic) [inline src/contribute/development-environment.md]:
```suggestion
    parent: contribute to eslint
```

**Review by mdjermanovic — COMMENTED** [review]
Some pages don't appear in the navigation because of non-existing parent keys.

**Comment by mdjermanovic** [comment]
https://github.com/eslint/eslint.org/pull/442 should be merged after this PR is merged.

**Comment by kecrily** [comment]
I'm going to write a script that automatically syncs keys that I always miss.

File: src/use/index.md [hunk 4]
```diff
@@ -1,34 +1,46 @@
...
 ESLint 命令行有许多标志，本章节对它们的作用进行了解释。
 
-## [基础](integrations)
+## [规则参考](../rules/)
+
+ESLint 有许多可以根据项目需求进行配置并微调的规则。此章节详细列出了所有规则并附上每一规则文档的链接。
+
+## [格式化工具参考](formatters)
+
+Control the appearance of the linting results with formatters. View all built-in formatters on this page.
```
Comment (aladdin-add) [inline src/use/index.md]: 这行可以删掉？

File: src/contribute/index.md [hunk 5]
```diff
@@ -16,34 +15,50 @@ eleventyNavigation:
...
 
 我们一直在寻找来自社区的贡献。本节解释了拉动请求的要求和贡献代码的过程。
 
+## [治理](governance)
+
+描述了 ESLint 的治理政策，包括个人在项目中的权利和特权。
+
+## [报告安全漏洞](report-security-vulnerability)
+
+要报告 ESLint 的安全漏洞，请使用我们的 [HackerOne 程序](https://hackerone.com/eslint)。
```
Comment (mdjermanovic) [inline src/contribute/index.md]: We're using GitHub advisories now, here's the original sentence in v8.33.0:

https://github.com/eslint/eslint/blob/13552c4e8d98c3b38b7dbff9c74dc82ca2aad385/docs/src/contribute/index.md?plain=1#L59

File: src/rules/no-multiple-empty-lines.md [hunk 6]
```diff
@@ -28,6 +28,7 @@ rule_type: layout
 
 var foo = 5;
 
+
```
Comment (mdjermanovic) [inline src/rules/no-multiple-empty-lines.md:31]: I think there should be another empty line here.

File: src/use/index.md (hunk 4 above)
Comment (kecrily) [reply src/use/index.md]: 不小心漏掉了

**Review by aladdin-add — APPROVED** [review]
LGTM.

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**